tests/MavenTests/testEIC.h

```diff
@@ -17,11 +17,9 @@ class TestEIC : public QObject {
 
     public:
         TestEIC();
+
     private:
-        const char* loadFile;
-        QStringList files;
-        QStringList files_ms2;
-        mzSample* _goodSample;
+        Samples* samples;
```

Comment:
IMO `Common` should be initialized once and should be accessible to everyone. 
It can be made global or we can use a singleton, there's a slight difference between them
Otherwise anyone who needs it will have to store a pointer of it, all the samples will have to be loaded again. I will have to do the same for Alignment 

What do you think ?

Reply:
How are you thinking of circulating the global (or singleton) `Samples` instance? I initially sought to do exactly what you wrote, but then I would have to declare `Samples` in `main.cpp` (for MavenTests module) or make `main.cpp` dependent on `common.h` and then pass the instance to each test upon instantiation. This seemed somewhat unclean to me. Like you said, loading the samples once for a test is not a costly operation, compared to loading them for each function (as it was before). I'm open to any good suggestions.

Reply:
We can have a file called `maventests.h`  with the following definition

```
...

#include "common.h"

namespace MavenTests 
{
    Samples* samples = new Samples;
    ....
    ....
}

....
```

And then in `testEic.cpp`

```
....
#include "maventests.h"


void TestEIC::testgetEICms2() {
    mzSample* sample1 = MavenTests::samples->ms2Samples[0];
    ....
} 

....
```

Do you know/think of other things that can be part of `MavenTests` ? If yes, then we can have something like this ?

Reply:
Maybe the database is also loaded every time 
Just making a guess

Reply:
@rish9511 Why the decision to create a separate `maventests.h` file? The common modules like samples and databases can be initialized in `common.h` as well. If it has the common functions for different test files, then the common modules like samples and databses should be part of it as well. Correct me if I'm missing something.

Reply:
> If it has the common functions for different test files, then the common modules like samples and databses should be part of it as well

Correct, this is also  what I realized later. The class `Common` can be used for all of this and maybe re-named to something better(MavenTests, TestUtils  etc)

Reply:
Great. I vote for TestUtils. I'm not sure if this change should be made in this particular PR, though. @saifulbkhan your call.

Reply:
I guess we will have to skip this one for now. 
@saifulbkhan  Do you think we will have time to refactor the `common` class ?

Reply:
I cannot guarantee I will have enough time for refactoring classes anymore. But since `common` is a relatively small class, I could give it a try.